src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+ 
   <body>
-    <h1>Search bar airbnb</h1>
+        
+    <form  action="https://mate-academy-form-lesson.herokuapp.com/create-application"  
+    method="post"  
+    data-qa="big"
+    class="form"
+    >  
+      <img 
+        src="images/Search.svg" 
+        alt="image"
+        class="searchimage"
+      >
+      <input 
+        type="text"
+        class="field field_focus"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     data-qa="small"
+     class="form form_little"
+     >
+      <img 
+        src="images/Search.svg"
+        alt="image"
+        class="searchimage searchimage_little"
+      >
+      <input 
+        type="text"
+        class="field field_little"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
```

Comment:
According to BEM methodology just change a syntax and naming a little
```suggestion
    <form 
       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
       method="post"
       data-qa="small"
       class="form form--small"
     >
      <img 
        src="images/Search.svg"
        alt="image"
        class="form__image form__image--small"
      >
      <input 
        type="text"
        class="form__input form__input--small"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```

Reply:
Not resolved. Check other cases.